tests/llsd_test.py

```diff
@@ -527,6 +527,26 @@ def testParseNotationHalfTruncatedHex(self):
     def testParseNotationInvalidHex(self):
         self.assertRaises(llsd.LLSDParseError, self.llsd.parse, b"'\\xzz'")
 
+    def testDeepMap(self):
+        """
+        Test formatting of a deeply nested map
+        """
+
+        test_map = {"foo":"bar", "depth":0}
+        max_depth = 199
```

Comment:
Would it make sense to set this from `MAX_FORMAT_DEPTH`?

Reply:
fixing